app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -454,6 +456,7 @@ public boolean onOptionsItemSelected(final MenuItem item) {
                 if (_document != null) {
                     _document.setFormat(itemId);
                     applyTextFormat(itemId);
+                    _appSettings.setDocumentFormat(getPath(), _document.getFormat());
```

Comment:
Moved all the setting of per-document options into the action code. This way we don't set per-document values unless the user actually sets them